code/modules/admin/admin_verbs.dm

```diff
@@ -494,19 +494,21 @@ var/list/admin_verbs_hideable = list(
 	set name = "Drop Bomb"
 	set desc = "Cause an explosion of varying strength at your location."
 
-	var/list/choices = list("Small Bomb", "Medium Bomb", "Big Bomb", "Custom Bomb")
-	var/choice = input("What size explosion would you like to produce?") in choices
+	var/list/choices = list("Small Bomb (1, 2, 3, 3)", "Medium Bomb (2, 3, 4, 4)", "Big Bomb (3, 5, 7, 5)", "Maxcap", "Custom Bomb")
+	var/choice = input("What size explosion would you like to produce? WARNING: These ignore the maxcap") in choices
```

Comment:
Still a chance the admin is not going to see this, that's why I suggested you place it as part of the max cap option, it's a less visually noisy area, their eyes have to look there to click it, etc.